runtime/bcverify/rtverify.c

```diff
@@ -1095,6 +1095,15 @@ verifyBytecodes (J9BytecodeVerificationData * verifyData)
 			for (i = 0; i < popCount; i++) {
 				if (type1 == BCV_GENERIC_OBJECT) {
 					type = POP;
+#if defined(J9VM_OPT_VALHALLA_VALUE_TYPES)
+					if (J9_ARE_ANY_BITS_SET(type, BCV_SPECIAL)
```

Comment:
Could this be moved into the else block at line 1133 ?

Reply:
What I mean is adding else block here (relative to `if (J9_ARE_NO_BITS_SET(type, BCV_SPECIAL)) {`)

https://github.com/eclipse-openj9/openj9/blob/d6334ad092bf544b470c350665359da53ad3c4fc/runtime/bcverify/rtverify.c#L1123